app/views/articles/_form.html.erb

```diff
@@ -1,4 +1,6 @@
 <%= form_with(model: article, class: "contents") do |form| %>
+    <script src="<%= asset_path("packs/tinymce.js") %>" type="module"></script>
```

Comment:
Why do we need an explicit script tag here? Does this work after you have deployed this feature?